### ISSUE TITLE
Return God Factions With Altered Benefits

### ISSUE THREAD

**Issue by cerulean-2** [Tslat/Advent-Of-Ascension#3787]
Apparently the Nevermine gods Luxon, Selyan, Erebon, and Pluton were removed mainly due to their buffs being overpowered. So, I suggest bringing back god factions entirely, but with different benefits in order to prevent power imbalance.

First, each dimension is usually ruled by the specific god that ruled it in earlier versions. Some dimensions could potentially have multiple gods ruling it, but for now I will mostly follow the model from earlier versions. To be explicit, these are the dimensions in each faction. (There are a couple tweaks to how it was before.)

Luxon rules the Haven, Runandor, the Shyrelands, and Candyland. These are all light-hearted and "happy" dimensions.

Selyan rules Creeponia, Lelyetia, Precasia, Gardencia, Mysterium, and L'Borean. These are all nature-themed places or are otherwise dense with foliage.

Erebon rules the Abyss, Vox Ponds, Dustopia, Greckon, the Nether, and Celeve. These are all evil and dark places.

Pluton rules Barathos, the Deeplands, Crystevia, Iromine, and Lunalus. These are all themed around the underground, metal, or treasure.

Nowhere has no god. The gods are fighting over the Overworld; the player can please all of them there. The End is left out as well.

Krasaun rules no dimensions because he is the elder god, so to speak, and he is not bothered with the quarrels of the lesser four.

When in a dimension ruled by a certain god, the player gets that god's buff. A god's buff can also be trained in the manners used in previous versions, but only in the Overworld (as before). Pleasing a god in the Overworld will grant that god's buff for the night only.

Essentially, the mechanics involving gaining a god's favor are unchanged.

However, the "buffs" that I speak of are not those we knew. There should be different benefits that are less overpowered.

One potential idea involves allowing easier training of certain skills. So, if a certain god is pleased (meaning his bar is completely full), the player will be able to train certain skills more quickly. The experience gained from training a skill will undergo a percent increase.

If the player pleases Luxon, he will be able to train Imbuing and Creation more quickly. Imbuing involves Pixons, which are made of light. Creation involves creating life. Likewise, for example, going to a place like the Haven will allow faster training since his bar is already full there. Convenient that Luxon has Pixons in all his dimensions.

If the player pleases Selyan, he will be able to train Farming, Faunamancy, and Fishing more quickly. All these involve nature or animals.

If the player pleases Erebon, he will be able to train Innervation and Dexterity more quickly. Innervation involves killing, and Dexterity is useful in combat.

If the player pleases Pluton, he will be able to train Engineering, Extraction, and Alchemy more quickly. All these involve either finding treasure or creating out of metal.

To please Krasaun, all gods will need to have their bars halfway full. This might happen in the Overworld. Then the player will be able to train all ten skills more quickly for the following night, but the increase will only be half that of a buff from one of the other gods. For example, the training increase for Innervation derived from going to the Abyss is twice that of the training increase derived from pleasing Krasaun in the Overworld.

There might be other ideas for the implementation of god buffs, but this was the first that came to mind. The god factions were one of the things that made AoA distinctive, and this is a way of returning them without the negative effects. The training increases do not need to be overpowered; each will be a percent increase that can be tinkered with before the mod's release.

**Comment by Cruuk** [comment]
It wasn't just removed for being OP. https://adventofascension.fandom.com/wiki/User:TheGreatCrackerz/Player_questions

![image](https://github.com/Tslat/Advent-Of-Ascension/assets/38159897/c14b0d2a-2139-4393-b694-b91f84dde77c)

A bunch of this seems to be shoehorning things into slots to fullfill a personalized interpretation, something that Tribute and the gods seem to attract a lot of. Faunamancy associated with the nature god? The patchouli description implies more sinister life manipulation. Hauling associated with the nature god? You can get lots of treasure boxes - what about the treasure god? We don't even know if Pixons will still be around. This arbitrary thematic slotting is one of the factors on why this system has proven to be unsalvageable.

This just seems to boil down to bringing it back for the sake of bringing it back, heavily based on current conventions. It's important to think more outward and assume anything can change. I don't see what this would add. We already have a way to boost exp with Utopian armor - this situational exp boost would be worthless going after if underpowered, or centralizing. In the past I've really tried suggesting reworks for tribute, and none of them worked out.

That thing with describing things as distinct or unique is very non-descriptive. Those vague phrases don't describe on why it was distinct. What made it distinct? The pot effects? The thematics? And if we were defined by this whole system, being defined by a poorly made system with poorly planned out lore doesn't sound like a good thing

**Comment by cerulean-2** [comment]
This suggestion can be tinkered with, of course.

There can be less reliance on the Overworld by allowing any god to be pleased in any dimension, though a certain god will automatically be pleased. Leaving the dimension will reset the Tribute bars. Therefore, Luxon will start off fully pleased in the Haven, though other gods could be given tribute as well. Luxon might even lose tribute there if mobs are killed for Erebon. My suggestion can be adapted in this way to allow more time in other dimensions.

As for my personal interpretations, somebody has to interpret the content of this mod at some point. Otherwise, it will remain vague forever. In other words, the gods can't be used if we're afraid to flesh out what they actually do. If Tslat decided that a skill belonged to a certain god, it wouldn't be so arbitrary, would it?

Obviously, a few of the skills could be swapped around between gods. I'd have to see what happens with Faunamancy before I can say that it should be Erebon's, but there would be no issue if that were the case. Why would Pixons be gone? Tslat went to all the trouble of making Imbuing recipes with all the Power Stones, and it looks like the new Imbuing skill keeps this. Even if Pixons were removed (I hope not), that's only one skill needing tweaking.

The bonuses can be reasonably useful, and they can stack with Utopian Armor. There might be additional benefits that Tribute can provide, and I will put forth further ideas about that as I conceive them.

Why were the gods "unique?" Well, there was nothing like this in other mods. Therefore, it would be a shame to scrap such a novel idea. Nevermine was not just an exploration mod. There was mystery, an element beyond what the player saw with his own eyes. There were beings beyond his experiences, who were watching over him as he danced around enemies and chasms in the Haven, suffered in Dustopia, and triumphed into the Shyrelands. This is a great opportunity for lore, and we're passing it up. This suggestion https://github.com/Tslat/Advent-Of-Ascension/issues/2441 is an example of the sort of stories that these four gods brought to the imagination. In fact, your quote, "to fullfill a personalized interpretation, something that Tribute and the gods seem to attract a lot of" shows that many people have put forth their own ideas about the gods. Lots of people are interested in the gods and want this mod to tell us more about them. We should respond to that demand and flesh out the gods more.

I hope this explains my "vague phrases" better. I don't want to go on these colorful tangents in my suggestions because, well, they're tangents. So I just say "unique" and continue with the suggestion.

The gods are _not_ "unsalvageable." I'd love to hear what your previous attempts were at reforming the Tribute system. They may provide inspiration for me in building upon my ideas, as I plan to do.

**Comment by Cruuk** [comment]
> Why would pixons be gone?

Think more outward than that. *Anything* can change, like how the power stones are obtained. Or even those Imbuing recipes. All that is confirmed about Imbuing is that its related to enchanting. That's it. Nothing else currently.

Nothing like this in other mods? I can think of some mods that add more ways to get pot effects and ones that add lore. Anyway:

> Nevermine was not just an exploration mod. There was mystery, an element beyond what the player saw with his own eyes. There were beings beyond his experiences, who were watching over him as he danced around enemies and chasms in the Haven, suffered in Dustopia, and triumphed into the Shyrelands

This is exactly what I mean by dancing around and obsessing over personalized headcanons. What I and other people see is a an MC mod with under-average dims. It's not being improved by twisting it into a made-up story with no substance. They weren't watching you, it was text in a screen with a pot effect. It is frankly obnoxious and has been a problem.

First, that git post you linked is an april fools post mocking the people that do this.

Second, the reason it's an issue is because we've had people make up their own personalized interpretations and attempt to apply them to actual discussion on why it's good lore... When it isn't. We're not passing it up, we're trying to get away from it.

Their "implementation" was nothing more than textual references and added nothing good to the mod beyond lore-bait and an obtuse system. What I mean by that is that the gods only existed to quickly hype people with no actual substance. We have the rest of the old lore from the old dev, and it is very barebones and has just as little substance:

Visualent was a spy of Erebon, when absolutely nothing hinted towards this. It is unknown if this was even planned at the time.

The Eeos lived in this place called "Tyfellnior", but Erebon attacked it 1 day, but they were saved by Krasaun, who made Mysterium for them. Even though Selyan is supposed to rule Mysterium, and why would a "god of equality" interfere like this? What makes the Eeos so important to interfere? How much of this was really thought out?

Erebon was gonna have a "right hand man" named the Shadowhands. Where does Penumbra fit into this though? Isn't he supposed to be the guardian? Was Penumbra even remembered?

Are you seeing a pattern here? It's style over substance. It's the equivalent of shallow fun-fact trivia - something that only exists to get a quick response. The old dev even admitted that he never even had a design thought of for Pluton and regretted him. He doesn't remember most of the mod in general, and does regret various decisions made.

The inner community has a distaste for the old hype culture generated by old teasers, and the old lore falls into that same group, because the 4 gods and any semblance of a "story" only existed as a hypothetical to spice up teasers.

And even if we did attempt to salvage it... What's the advantage of it? The thing people like to interpret beyond one-note text and pot effects is that there's these 4 guys for each aspect of the world, and 1 equal guy. As I've said, it has proven difficult to arbitrarily slot things to fit into these 4 gods. Trying to cohesively fit everything into archetypes of good/evil/nature/fortune while trying to define how much those concepts can expand would be a design hassle, and would take away from parts of the mod that need actual improving.

And don't assume I'm some evil no-fun guy. New lore is being made as the mod is reworked. We already have a few named characters: Krasaun is still around according to worn book, though we don't know what his new purpose will be. There's also the "Arc-ancestors" in the torn pages entry.

It's not much at all right now, but it's something people can still make personalized headcanons over in the future as we get more stuff. My point is that I don't think we're against the idea of having lore that incentivizes people interpreting an MC mod as "There was mystery, an element beyond what the player saw with his own eyes.", it's that there is no interest and dislike of the old 5 gods system from an implementation and conceptual standpoint, even ignoring the gameplay issues of the gameplay feature it's correlated with. I'd say Nowhere's lore already provides a similar "mysterious" feel with room for interpretation, but that's just me.

Theoretically there's nothing stopping the lore of going in a similar direction of powerful beings that rule over dims, but remember: Gameplay comes first. Time spent on lore is time not spent on gameplay. Trying to morph the mod around for the sake of some divination-esque system is something I don't think can really work out. It would be a lot of work to implement it well, when we still have the dims and 5 remaining skills to worry about

**Comment by cerulean-2** [comment]
Not to argue endlessly, but this _is_ reasonably interesting.

You're right that many elements of the mod can change. If that means that this suggestion doesn't fit anymore, then I will alter it. Until then, there's no issue. I mean, why suggest anything if none of the mod can be relied on as existent?

**There were never any potion effects in this suggestion.** Part of the point was suggesting a replacement to simple potion effects. There are only skill training boosts in this suggestion.

I did not create this suggestion to put forth my storyline for the mod's lore. I wrote the suggestion in order to propose **actual game mechanics** that can be put in place. This is what you too seem to want. I only went into lore because you wanted me to explain in detail why I thought the gods were worth keeping. Part of it is the potential for lore, no doubt, but there is also the potential for interesting game mechanics based on the god factions.

If the mod is lacking substance, then here are some new mechanics that will help to remedy that. The thematics behind the mechanics? I've justified that too.

Now, if everyone is trying to get away from headcanons, well, the gods were a pretty indisputable part of the mod before. Xolova made them up, not I. It's not like the gods are some topic that exists in some headcanons but not others. An official canon can exist with them included very easily. The new and upcoming bits of lore will be perfectly ample for justifying the inclusion of the gods. Tslat can improve the simple and hype-focused lore from before while fixing any inconsistencies.

As you say, the mechanics are more important than the lore. Indeed, this suggestion was not supposed to be about the lore; it's supposed to be about the mechanics that I have proposed. Yet you at the same time seem quite concerned with the lore backstory of any new mechanics.

What do you dislike? Do you dislike that Xolova made hype-focused and simple lore while putting priority on mechanics? Would you rather he had made better lore and ignore the mechanics? After all, time spent on lore is not spent on mechanics, as you say.

I appreciate the time that you have taken to respond to my thoughts in a comprehensive manner. However, I feel that when I try to remedy the lore problems, you say that I should be focusing on the mechanics. When I try to defend my proposed mechanics, you criticize the lore behind them. Your arguments seem at least a little inconsistent to me.

**Comment by Cruuk** [comment]
> Not to argue endlessly, but this _is_ reasonably interesting.
> 
> You're right that many elements of the mod can change. If that means that this suggestion doesn't fit anymore, then I will alter it. Until then, there's no issue. I mean, why suggest anything if none of the mod can be relied on as existent?
> 
> **There were never any potion effects in this suggestion.** Part of the point was suggesting a replacement to simple potion effects. There are only skill training boosts in this suggestion.
> 
> I did not create this suggestion to put forth my storyline for the mod's lore. I wrote the suggestion in order to propose **actual game mechanics** that can be put in place. This is what you too seem to want. I only went into lore because you wanted me to explain in detail why I thought the gods were worth keeping. Part of it is the potential for lore, no doubt, but there is also the potential for interesting game mechanics based on the god factions.
> 
> If the mod is lacking substance, then here are some new mechanics that will help to remedy that. The thematics behind the mechanics? I've justified that too.
> 
> Now, if everyone is trying to get away from headcanons, well, the gods were a pretty indisputable part of the mod before. Xolova made them up, not I. It's not like the gods are some topic that exists in some headcanons but not others. An official canon can exist with them included very easily. The new and upcoming bits of lore will be perfectly ample for justifying the inclusion of the gods. Tslat can improve the simple and hype-focused lore from before while fixing any inconsistencies.
> 
> As you say, the mechanics are more important than the lore. Indeed, this suggestion was not supposed to be about the lore; it's supposed to be about the mechanics that I have proposed. Yet you at the same time seem quite concerned with the lore backstory of any new mechanics.
> 
> What do you dislike? Do you dislike that Xolova made hype-focused and simple lore while putting priority on mechanics? Would you rather he had made better lore and ignore the mechanics? After all, time spent on lore is not spent on mechanics, as you say.
> 
> I appreciate the time that you have taken to respond to my thoughts in a comprehensive manner. However, I feel that when I try to remedy the lore problems, you say that I should be focusing on the mechanics. When I try to defend my proposed mechanics, you criticize the lore behind them. Your arguments seem at least a little inconsistent to me.

My PC broke, but now it's fixed and I can post again.

Most of that was tangential on the old god stuff itself. Though I did it so I could use the same tangent in the future for other purposes.

I apologize, to prevent this from being endless I will focus on the absolute root of the suggestion itself. So, you wish to add exp boosts correlated with the old 4 gods, yes? If so, the main issues I have with that are:

1. We already have a partially-challenging to get 30% xp boost from Utopian - its defining characteristic. The thing with these types of mechanics is that you have to find a perfect inbetween in a way that isn't redundant. If it gives too little, no player will go after it and it will be no different from no xp boost. If it gives too much, it would be centralizing. E.G. even if it was easy and have a +10% xp boost, it wouldn't be worth it to go after due to that non-difference. It's a factor of giving it a point and how it compares to similar abilities like Utopian. 

Even if they stacked, it doesn't change that people would just use Utopian instead or use this instead. It'd be excessive to use both. Utopian is already a bonus that isn't *really* needed unless you want speedy xp, and more exp boosts wouldn't really change this and make things worse in this regard. adding more xp boosts won't change *how* many xp boost items are used.

The skill helms are moreso there to help with levelling beyond 100, so they're not a factor here. In other words, it's complicated.

You haven't how you would get these boosts exactly, so I can't examine further.

2. Situational exp boosts onto specific groups of skills isn't an inherently bad idea. But even if we did implement it, there are probably better ways to implement it than trying to involve the old scrapped gods. I don't think it would be an ideal way to implement it.

First, there's the fact that each god is correlated with a different amount of skills which creates some inequality to me (if the thematic idea of it according to people is that there's these symmetrical gods with a 5th equal guy). And second, there's the thing of it being a hassle of trying to work with the thematic specifics of light/dark/nature/fortune or whatever. E.G. the thing I said earlier where it's weird to correlate hauling with Selyan when that has lots of treasure boxes when a "fortune" god exists.

Basically, trying to shoehorn the old gods into it is an unideal way to implement other forms of xp boosts

**Comment by PraseodymiumSpike** [comment]
Honestly, I never liked the God system.
